.npmignore

```diff
@@ -1 +1,2 @@
 !*.js
+test
```

Comment:
I disagree on excluding tests from npm. I prefer files such as tests or source files to be available to users. This does make node_modules slightly larger, but is more convenient when inspecting source or debugging downstream modules.